java/core/page-factory-utils/src/main/java/ru/sbtqa/tag/pagefactory/util/PageFactoryUtils.java

```diff
@@ -117,6 +115,13 @@ public static Boolean isRequiredAction(Method method, final String title) {
         
         return false;
     }
+
+    private static I18N getI18nByCurrentScenario(Method method) {
```

Comment:
its wrong name and dirty signature. Refactor it like
`getI18nByCurrentScenario(Class callerClass, Scenario scenario)`

And use it as 
`getI18nByCurrentScenario(method.getDeclaringClass(), ScenarioContext.getScenario())`
It will make this method more clean and stateless.
Also, it can be moved to qaUtils in this shape.

Reply:
Ok, I'll move it to qa-utils I18N